### PR TITLE
Add AUR package

### PR DESCRIPTION
**Pull request by jneidel**
I created an AUR package and added it accordingly.

https://aur.archlinux.org/packages/fjira

### CODE REVIEW

**Comment by mk-5** [comment]
I will check why goreleaser check is failing here ... it shoudn't , it's just README change

**Comment by jneidel** [comment]
Actually if you're using goreleaser we can also do it through there.

I'll make that adjustment tomorrow and create the account where you can register your SSH key for you tomorrow.

(ref: https://github.com/Yakitrak/obsidian-cli/pull/17)

**Comment by mk-5** [comment]
> Actually if you're using goreleaser we can also do it through there.
> 
> If you want, I'll make that adjustment tomorrow and create the account where you can register your SSH key for you.
> 
> (ref: [Yakitrak/obsidian-cli#17](https://github.com/Yakitrak/obsidian-cli/pull/17))

I will have a look

**Comment by jneidel** [comment]
@mk-5 I added the goreleaser config.

I've sent you the credentials to your new AUR account via email.

You can setup your own keypair for pushing there :)

**Comment by mk-5** [comment]
I will have a look soon

**Comment by mk-5** [comment]
@jneidel can you rebase? I changed something in CI workflows in order to make them run for forked branches

File: .goreleaser.yaml [hunk 1]
```diff
@@ -95,6 +95,19 @@ chocolateys:
     api_key: '{{ .Env.CHOCOLATEY_API_KEY }}'
     source_repo: "https://push.chocolatey.org/"
     goamd64: v1
+    
+aurs:
+  - name: fjira-bin
+    homepage: https://github.com/mk-5/fjira
+    description: Fjira - the fuzziest Jira command line tool in the world
+    maintainers:
+      - 'Jonathan Neidel <aur at jneidel dot com>'
```
Comment (mk-5) [inline .goreleaser.yaml:104]: can you add me as a second maintainer here?

- `Mateusz Kulawik <mateusz+aur@mk5.pl>`

**Comment by jneidel** [comment]
Done :smile:

**Comment by mk-5** [comment]
> Rebased 😄

okay there is one failure about commits message. The second one with Goreleaser - Check / it's something I need to fix - but I can ignore that check for now. There is a problem with some of the secrets that are loaded for that workflow. I don't know why (yet), but it doesn't work when PR is coming from forked repos.

**Comment by jneidel** [comment]
Rebased :smile:

**Comment by mk-5** [comment]
> Rebased 😄

all right 👍 I will try to take this up soon. I'm not so into AUR Packages, I'd like to test it locally before I'll merge it :)

**Comment by jneidel** [comment]
Sure, just let me know here when you released, then I'll check that the pkg looks good in the AUR :smile:

**Comment by mk-5** [comment]
@jneidel locally "it works" 😄  I'll try to release 1.1.2 version with AUR in a minutes

**Comment by mk-5** [comment]
@jneidel it doesn't work - I think I found a reason - https://goreleaser.com/customization/aur/

```
    # The package name.
    #
    # Note that since this integration does not create a PKGBUILD to build from
    # source, per Arch's guidelines.
    # That said, GoReleaser will enforce a `-bin` suffix if its not present.
    #
    # Default: ProjectName with a -bin suffix.
    name: package-bin
```

Goreleaser is forcing package to have `-bin` suffix. I just changed it - we will see

**Comment by mk-5** [comment]
@jneidel okay - it is here https://aur.archlinux.org/packages/fjira-bin

would be nice if you can remove that old package https://aur.archlinux.org/packages/fjira

**Comment by jneidel** [comment]
@mk-5 I have requested the `fjira` package to be deleted.

Feel free to add me as a co-maintainer to the `fjira-bin` package in the AUR.

**Comment by mk-5** [comment]
> @mk-5 I have requested the `fjira` package to be deleted.
> 
> Feel free to add me as a co-maintainer to the `fjira-bin` package in the AUR.

you are in a `maintainers` list in goreleaser.yml 

image

mhmm, I will check it from AUR panel side

**Comment by mk-5** [comment]
> > @mk-5 I have requested the `fjira` package to be deleted.
> > Feel free to add me as a co-maintainer to the `fjira-bin` package in the AUR.
> 
> you are in a `maintainers` list in goreleaser.yml
> 
> image
> 
> mhmm, I will check it from AUR panel side

okay - I've just added you via Panel

image

**Pull request merged**